src/main/java/net/sf/jabref/model/database/KeyCollisionException.java

```diff
@@ -25,20 +25,43 @@
 */
 package net.sf.jabref.model.database;
 
+/**
+ *
+ * Represents an Exception that will be thrown if a key will be insert in a Collection,
+ * but the same key exists in the Collection.
+ * @see {@link java.lang.RuntimeException}
+ *
+ */
 public class KeyCollisionException extends RuntimeException {
 
+    /**
+     * Constructor for a KeyCollisionException without any parameters.
+     */
     public KeyCollisionException() {
         super();
     }
 
+    /**
+     * Creates a KeyCollisionException with a given message.
+     * @param msg   the message to be shown, when throwing this exception
+     */
     public KeyCollisionException(String msg) {
         super(msg);
     }
 
+    /**
+     * Creates a KeyCollisionException with a given message and a given cause.
```

Comment:
I don't think, it necessary to document trivial cases. Maybe, this teamscale finding should be ignored?